cgmes/cgmes-model/src/main/resources/CIM16.sparql

```diff
@@ -826,8 +826,7 @@ WHERE {
         cim:NonlinearShuntCompensatorPoint.NonlinearShuntCompensator ?Shunt ;
         cim:NonlinearShuntCompensatorPoint.sectionNumber ?sectionNumber ;
         cim:NonlinearShuntCompensatorPoint.b ?b ;
-        cim:NonlinearShuntCompensatorPoint.g ?g  .
-    FILTER REGEX ( STR (?Shunt), "{0}")
+        cim:NonlinearShuntCompensatorPoint.g ?g ;
```

Comment:
minor syntax thing: predicate-object lists should end in a dot (or nothing, if it is the last triple pattern in the group)

Reply:
fixed